### ISSUE TITLE
Text is not extracted from links

### ISSUE THREAD

**Issue by Unrud** [markdownlint/markdownlint#405]
## Description
Text is not extracted from links.

### Environment

**MDL Version**
[v0.11.0](https://github.com/markdownlint/markdownlint/tree/v0.11.0) and [master](https://github.com/markdownlint/markdownlint/tree/0b006db2722db937dfdf1d842c1b410b69d3e41f)

### Expected Behavior

```markdown
> <http://example.org> more text.
```

The example above doesn't violate the rule [MD027 - Multiple spaces after blockquote symbol](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md027---multiple-spaces-after-blockquote-symbol).

### Actual Behavior
**markdownlint** thinks the example violates MD027.

Text from links is ignored:

https://github.com/markdownlint/markdownlint/blob/0b006db2722db937dfdf1d842c1b410b69d3e41f/lib/mdl/doc.rb#L255-L263

Just adding `a` to the list solves the problem for MD027 but causes issues with other rules because they rely on text from links being ignored (?).

**Comment by jaymzh** [comment]
This is fixed in 0.16.0
